tests/fp8/test_fp8_parameter.py

```diff
@@ -0,0 +1,25 @@
+import torch
+from nanotron.fp8.constants import DTypes
+from nanotron.fp8.meta import FP8Meta
+from nanotron.fp8.parameter import FP8Parameter
+from nanotron.fp8.tensor import FP8Tensor
+
+
+# @pytest.mark.parametrize("quantize_input", [True, False])
+def test_create_fp8_parameter():
+    tensor = torch.randn(16, 16, device="cuda", dtype=torch.float32)
+    # tensor = FP8Tensor(tensor, DTypes.FP8E4M3) if quantize_input else tensor
```

Comment:
can we keep this?

Reply:
there are a few issues. will fix it in another PR!! 

https://github.com/huggingface/nanotron/blob/715bf38dd2c82f136e94e25f7e32e440bee7967c/src/nanotron/fp8/parameter.py#L22